docs/CHANGELOG.rst

```diff
@@ -32,6 +32,7 @@ Fixed
   relevance
 - Support filtering by type on knowledge base Feature API
 - Attach ``rose2`` Data object to the input sample
+- Fix enrichment heatmap plot in ``chipqc`` process
```

Comment:
The change is just a change of Docker image version. Perhaps this comment should reflect (only) that?